.github/workflows/rust-clippy.yml

```
@@ -38,4 +38,4 @@ jobs:
       - name: Run cargo clippy
         working-directory: ${{ inputs.working-directory }}
         run: |
-          cargo clippy ${{ env.LOCKED_FLAG }} -- -D warnings
+          cargo clippy ${{ env.LOCKED_FLAG }} --all -- -D warnings
```

```suggestion
          cargo clippy ${{ env.LOCKED_FLAG }} --workspace -- -D warnings
```
Let's actually use `--workspace`, cargo check CLI helps says:
`-all            Alias for --workspace (deprecated)`

---

Ah, thanks.